### ISSUE TITLE
Cant delete settings

### ISSUE THREAD

**Issue by TediPapajorgji** [facebookincubator/facebook-for-woocommerce#481]
Hi,

Trying to delete settings by going to "advanced options" then pressing "delete settings" but it is giving me the error "Sorry, something went wrong. This might be a temporary issue and we're working to get it fixed as soon as we can. Please try again later."

If you can help me sort this issue out it would be appreciated.
Our website is https://newlinkz.com/
Pixel Id: 1831879040447647

**Comment by sammul** [comment]
@TediPapajorgji are you admin on the page?

**Comment by TediPapajorgji** [comment]
@sammul I have made myself a page admin, thanks for your support. It now seems as the woocommerce pixel is stuck in a weird state. On the popup configuration page for facebook for woocommerce, it says it is setup for the pixel with ID 1831879040447647 but in reality its actually reporting to another pixel with id 889308007869481, I determined this from the analytics page in facebook business manager. I tried to switch from pixel ID 1831879040447647  to 889308007869481 earlier this week but it stayed stuck in a loading screen forever that's why i attempted to reset the pixel data completely.
